docs/assets/js/charts.js

```diff
@@ -429,7 +429,7 @@ function createHistoryChart(canvas, actionBar)
 
                     // The date range is of the form [t0, t1), inclusive of t0 but exclusive of t1.
                     // Hence, subtract one second from t1 to obtain the previous date in UTC
-                    let t1 = dateRange[1];
```

Comment:
Do you see an easy way to cover that with a test case? E.g. move `tooltipTitleCallback` to a "real" function and then test this function with some demo data?

Reply:
I thought about this as well, but now that you think of unit-testing this as well, I probably should do this tomorrow :slightly_smiling_face:.